src/Compilers/Core/Portable/Compilation/NullableContext.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents the state of the nullable analysis at a specific point in a file. Bits one and
+    /// two correspond to whether the nullable feature is enabled. Bits three and four correspond
+    /// to whether the context was inherited from the global context.
+    /// </summary>
+    [Flags]
+    public enum NullableContext
+    {
+        /// <summary>
+        /// Nullable warnings and annotations are explicitly turned off at this location.
+        /// </summary>
+        Disable = 0,
```

Comment:
seems like it should be past tense "Disabled"

Reply:
This matches the text the user will type in the directive.

---
In reply to: [303151585](https://github.com/dotnet/roslyn/pull/37017#discussion_r303151585) [](ancestors = 303151585)

Reply:
Isn't this the state between the `#nullable enable` and the next nullable directory rather than at the directive?

---
In reply to: [303190777](https://github.com/dotnet/roslyn/pull/37017#discussion_r303190777) [](ancestors = 303190777,303151585)

Reply:
I view it more as the effect of the `#nullable` directive, and should match the name. @agocke, thoughs?

---
In reply to: [303191060](https://github.com/dotnet/roslyn/pull/37017#discussion_r303191060) [](ancestors = 303191060,303190777,303151585)

Reply:
I still think this should be past tense "Disabled". We're describing the state at a point in the source

---
In reply to: [303616765](https://github.com/dotnet/roslyn/pull/37017#discussion_r303616765) [](ancestors = 303616765,303191060,303190777,303151585)

Reply:
Hmm, I had suggested "Disable" but it does seem like they're describing different things. One's describing the state ("Disabled"), while the other's describing the option ("Disable"). The adjective form seems appropriate here.